Week2/homework/js-exercises/convertGrades.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+let gradeUsa = '';
+
+let score ='';
```

Comment:
You can move `gradeUsa` inside the body of the `function`. This keeps the variables encapsulated inside which makes the function much more reusable. Also, you do not need to declare `score` at all -- the argument `score` already is visible inside the body of the function.